gatsby-config.js

```diff
@@ -55,6 +55,7 @@ module.exports = {
                 ],
                 serialize: ({ site, allSitePage }) =>
                     allSitePage.edges.map((edge) => {
+                        const ignore_localized_regex = /careers/;
```

Comment:
```suggestion
                        const careers_regex = /careers/;
```

Reply:
Maybe we can call this by its name perhaps? And then using it in `ignore_localized`

Reply:
Was planning on using this to probably add more pages into this list in future